.github/workflows/pr.yml

```
@@ -23,3 +23,51 @@ jobs:
         with:
           image: ${{ env.IMAGE_NAME }}
           args: --file=Dockerfile --policy-path=./.snyk --severity-threshold=high --exclude-base-image-vulns --org=dwp-dataworks
+
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+
+      - name: Run tests
+        run: |
+          pip install -r requirements.txt
+          python -m pytest -v
```

I'd just comment this out, then if and when needed it can be uncommented or changed to `tox` or `make` commands
```suggestion
          #python -m pytest -v
```